platform/android/export/export.cpp

```diff
@@ -2016,6 +2018,13 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			ret = unzGoToNextFile(pkg);
 		}
 
+		if (!invalid_abis.empty()) {
+			String unsupported_arch = String(", ").join(invalid_abis);
+			EditorNode::add_io_error("lib for the selected architectures" + unsupported_arch + "is not found in the export template.\
```

Comment:
Seems you may need a space before `"is not found..."`.

Reply:
Yea my bad I was having trouble compiling for Android on master branch so I didn't test it. 
This time I changed the line and tested it on 3.2
![androidExportValidation](https://user-images.githubusercontent.com/23119524/78086057-15348c80-73bd-11ea-909d-0f46cb22dd98.png)